inductiva/tasks/streams.py

```diff
@@ -136,9 +151,13 @@ def _ansi_message_formatter(self, msg):
         # 5. move down SPACING lines
         # 6. move to beginning of line and clear line
         # 7. add new blank line
-        return f"{UP}\r{CLEAR_LINE}{msg}{DOWN}\r{CLEAR_LINE}\n"
+        if is_stderr:
+            m = f"{UP}\r{CLEAR_LINE}{FAILURE}{msg}{RESET}{DOWN}\r{CLEAR_LINE}\n"
+        else:
+            m = f"{UP}\r{CLEAR_LINE}{msg}{DOWN}\r{CLEAR_LINE}\n"
+        return m
 
-    def _plain_message_formatter(self, msg):
+    def _plain_message_formatter(self, msg, **_):
```

Comment:
Could you add a simple comment to explain why the **_ ?

Reply:
"# _ to make it have the same number or arguments as the other formatters
 # In this case, the second argument is not used."

Done. Is this clear ?

Reply:
Yes! Thanks :)